cv32e20/tests/programs/custom/interrupt_test/interrupt_test.c

```diff
@@ -272,7 +272,8 @@ int main(int argc, char *argv[]) {
     if (retval != EXIT_SUCCESS)
         return retval;
 
-    // Repeat test1 (restore vector mode)
+// unused tests. to be removed at clean-up 
```

Comment:
Recommend adding the string "TODO" in this comment.  Most editors will automatically highlight it and it also makes it easier to grep code to find what needs to be cleaned-up.